src/v3/src/transformer/oktaVerify/transformOktaVerifyChallengePoll.ts

```diff
@@ -46,7 +46,7 @@ export const transformOktaVerifyChallengePoll: IdxStepTransformer = (
     } as TitleElement);
 
     const resendStep = availableSteps?.find(({ name }) => name?.endsWith('resend'));
-    if (canResend && resendStep) {
```

Comment:
Same here, are `canResend` and the existence of the step in availableStep always in synch?